### PR TITLE
refactor create / find / delete end points for cards

### PR DESCRIPTION
**Pull request by eoinmcafee00**
Introduce route to persist card

### CODE REVIEW

File: operator/manager/rpc2/handler.go [hunk 1]
```diff
@@ -26,6 +26,8 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/drone/drone/handler/api/render"
```
Comment (marko-gacesa) [inline operator/manager/rpc2/handler.go:29]: Organize imports. Move this line below, where the other drone/drone imports are.

File: operator/manager/rpc/client.go [hunk 2]
```diff
@@ -38,6 +39,10 @@ type Client struct {
 	client *retryablehttp.Client
 }
 
+func (s *Client) UploadCard(ctx context.Context, step int64, input *core.CardInput) error {
```
Comment (tphoney) [inline operator/manager/rpc/client.go:42]: this should have a test similar to the other functions in this package.

**Review by tphoney — CHANGES_REQUESTED** [review]
squash the commits and have a look at the changes in go.mod

File: operator/manager/rpc2/handler.go (hunk 1 above)
Comment (eoinmcafee00) [reply operator/manager/rpc2/handler.go:29]: sorted

File: operator/manager/rpc/client.go (hunk 2 above)
Comment (eoinmcafee00) [reply operator/manager/rpc/client.go:42]: To write tests similar to the ones in the package, id have to write code to hook it up in server.go which is kinda pointless dead code?

**Review by tphoney — APPROVED** [review]

**Pull request merged**